### PR TITLE
build: update versioneer

### PR DESCRIPTION
**Pull request by anjakefala**
- we need versioneer at least 0.19 to have get_cmdclass accept an argument

See: https://github.com/python-versioneer/python-versioneer/issues/161#issuecomment-693070731

### CODE REVIEW

File: caput/_version.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
     """
     rootdirs = []
 
-    for _ in range(3):
```
Comment (nritsche) [inline caput/_version.py:119]: `i` is still not used here, so this will trigger some linter warnings.

File: caput/_version.py [hunk 2]
```diff
@@ -241,7 +241,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
     if sys.platform == "win32":
         GITS = ["git.cmd", "git.exe"]
 
-    _, rc = run_command(GITS, ["rev-parse", "--git-dir"], cwd=root, hide_stderr=True)
```
Comment (nritsche) [inline caput/_version.py:244]: Same as above

File: caput/_version.py [hunk 3]
```diff
@@ -524,7 +524,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for _ in cfg.versionfile_source.split("/"):
```
Comment (nritsche) [inline caput/_version.py:527]: Same as above

**Review by nritsche — CHANGES_REQUESTED** [review]

**Review by nritsche — APPROVED** [review]
:+1:

File: caput/_version.py (hunk 1 above)
Comment (anjakefala) [reply caput/_version.py:119]: I saw that! I want to ignore the linter warnings for this file, but would you prefer for me to modify `_version.py`?

**Pull request merged**